UnityProjectDP/Assets/Scripts/Visualization/Animation/AnimationData.cs

```diff
@@ -27,12 +27,16 @@ public List<Anim> getAnimList()
         public void AddAnim(Anim anim)
         {
             animations.Add(anim);
+            GameObject.Find("MainPanel").transform.Find("Edit").GetComponentInChildren<Button>().interactable = true;
+            GameObject.Find("MainPanel").transform.Find("Play").GetComponentInChildren<Button>().interactable = true;
```

Comment:
Toto mozno vytiahnut niekam do UI kazde do svojej metody s atributom pre bool interactable. nieco v zmysle 
```suggestion
void EditButtonInteractable(bool interactable)
{
GameObject.Find("MainPanel").transform.Find("Edit").GetComponentInChildren<Button>().interactable = interactable;
 }
```
pripadne
```suggestion
void SetButtonInteractable(bool interactable, string buttonName)
{
GameObject.Find("MainPanel").transform.Find(buttonName).GetComponentInChildren<Button>().interactable = interactable;
 }
```